src/caja-desktop-window.c

```diff
@@ -204,6 +274,9 @@ caja_desktop_window_new (CajaApplication *application,
         XFree(xch);
     }
 
+    gtk_widget_hide (CAJA_WINDOW (window)->details->menubar);
+    gtk_widget_hide (CAJA_WINDOW (window)->details->statusbar);
+
```

Comment:
this is not needed in the X11 case (that doesn't have the `show_all()`), is it?  Also, shouldn't you use `gtk_widget_show()` above instead?

Reply:
gtk_widget_show() works as suggested, and keeps the statusbar off the desktop.